node/parachain/src/chain_spec.rs

```diff
@@ -98,6 +98,12 @@ fn testnet_genesis(
 			accounts: BTreeMap::new(),
 		}),
 		pallet_ethereum: Some(EthereumConfig {}),
-		stake: Some(StakeConfig { stakers: vec![] }),
+		stake: Some(StakeConfig {
+			stakers: endowed_accounts
+				.iter()
+				.cloned()
+				.map(|k| (k, None, 100_000))
+				.collect(),
+		}),
```

Comment:
> So all of the endowed accounts are the initial stakers.

Yes, that is correct.

> we have to give a particular CLI flag to include one of those accounts in the author inherent right?

I guess presumably just like the node CLI which now has the `--account-id` flag.

> Can you give an example of what the collator command would look like?

Collator command to do what? To join the collator set or to author blocks?

Reply:
Imagine a new user hears of moonbeam and wants to try it out. They decide to run a simple standalone node. So the run `moonbase-standalone --dev`. But now their chain doesn't work because no blocks meet the author inherent requirement. Doesn't it have to be `moonbase-standalone --dev --author-id ???`?

Reply:
I think this should go with the --dev/alice/bob work